src/i18n/locale-en.json

```diff
@@ -664,20 +664,6 @@
         "help.what.content": "Talk to Your Graph is chat bot based on OpenAI's GPT-4 model that can answer questions about data stored in GraphDB, leveraging both your data and the general knowledge of the GPT-4 model.",
         "help.how.title": "How does it work?",
         "help.how.content": "The bot uses the ChatGPT Retrieval connector to fetch information it does not know about. As such, you must first create a suitable connector. See also the settings for fine-tuning the integration with the Retrieval connector.",
```

Comment:
"Removes the comma at the end of the label. Apply the same change to all locale files."

Reply:
damn, I wondered why the build failed